packages/realm-server/server.ts

```diff
@@ -156,7 +156,7 @@ export class RealmServer {
       if (fileName.includes('.')) {
         ctxt.type = fileName.split('.').pop()!;
       } else {
-        ctxt.type = 'application/vnd.api+json';
+        ctxt.type = 'application/vnd.api+json'; // TODO: should we still default to this???
```

Comment:
I could use input on this @habdelra

Reply:
or another idea is that this logic for determining the mime type from the file extension could move into the realm  (right now it leans heavily on a koa feature--but i think that feature is available as the npm `mime-types` pkg)

Reply:
Updated.